core/templates/dev/head/components/sharing_links_directive.html

```diff
@@ -1,40 +1,76 @@
 <script type="text/ng-template" id="components/sharingLinks">
-  <ul class="oppia-sharing-links" layout="<[layoutType]>" layout-align="<[layoutAlignType]>">
+  <ul class="oppia-sharing-links" layout="<[layoutType]>" layout-align="<[layoutAlignType]>" explorationId="[getExplorationId]">
     <li>
-      <a ng-href="https://plus.google.com/share?url=<[serverName]>/explore/<[explorationId]>" onclick="return !window.open(this.href, '', 'height=600, width=600, menubar=no, toolbar=no, resizable=yes, scrollbars=yes')" ng-click="registerShareExplorationEvent('gplus')" target="_window">
+      <a ng-href="https://plus.google.com/share?url=<[serverName]>/explore/<[explorationId]>" onclick="return !window.open(this.href, '', 'height=600, width=600, menubar=no, toolbar=no, resizable=yes, scrollbars=yes');" ng-click="registerShareExplorationEvent('gplus');" target="_window">
         <img class="share-option-img" src="/images/general/gplus.png" alt="Google+">
       </a>
     </li>
 
     <li>
-      <a ng-href="https://www.facebook.com/sharer/sharer.php?sdk=joey&u=<[serverName]>/explore/<[explorationId]>&display=popup&ref=plugin&src=share_button" onclick="return !window.open(this.href, '', 'height=336, width=640')" ng-click="registerShareExplorationEvent('facebook')" target="_window">
+      <a ng-href="https://www.facebook.com/sharer/sharer.php?sdk=joey&u=<[serverName]>/explore/<[explorationId]>&display=popup&ref=plugin&src=share_button" onclick="return !window.open(this.href, '', 'height=336, width=640');" ng-click="registerShareExplorationEvent('facebook');" target="_window">
         <img class="share-option-img" src='/images/general/fb.png' alt="Facebook">
       </a>
     </li>
 
     <li>
-      <a ng-href="https://twitter.com/share?text=<[escapedTwitterText]>&url=<[serverName]>/explore/<[explorationId]>" onclick="return !window.open(this.href, '', 'height=460, width=640')" target="_window">
-        <img class="share-option-img" src="/images/general/twitter.png" ng-click="registerShareExplorationEvent('twitter')" alt="Twitter">
+      <a ng-href="https://twitter.com/share?text=<[escapedTwitterText]>&url=<[serverName]>/explore/<[explorationId]>" onclick="return !window.open(this.href, '', 'height=460, width=640');" target="_window">
+        <img class="share-option-img" src="/images/general/twitter.png" ng-click="registerShareExplorationEvent('twitter');" alt="Twitter">
       </a>
     </li>
 
     <li>
-      <a href="" ng-click="showEmbedExplorationModal(explorationId)" tooltip="Embed" tooltip-placement="top">
-        <i class="material-icons md-18" style="color: black;">&#xE157;</i>
+      <a href="" ng-click="showEmbedExplorationModal(explorationId); $event.stopPropagation();" tooltip="Embed" tooltip-placement="top">
```

Comment:
Hadn't before but related to my comment below. Removed now but in the dashboard, all of the sharing links were working, but the parent window went to the editor, instead of remaining on the dashboard page. Not an issue in the publish modal in the editor.